### ISSUE TITLE
Change Build System

### ISSUE THREAD

**Issue by Riptils** [nuIIpointerexception/vterm#5]
### Check for existing issues

- [X] Completed

### Describe the feature

I would like you to use Meson instead of makefile after transitioning from Just. Thank you, it would help me to create a solid PKGBUILD for Arch Linux. Thanks.

### If applicable, add some sort of preview to help us visualize your idea.

https://mesonbuild.com/

**Comment by nuIIpointerexception** [comment]
Alright, good suggestion.
Will take a look at this!

**Comment by Riptils** [comment]
Hi, I was wondering if just using cargo wouldn't be a better solution.

**Comment by nuIIpointerexception** [comment]
Yeah, I also thought the same thing, i think using cargo build --release is the correct approach. I will most likely just use the build.rs to attach manifests and icons for platforms and embed the resources into the app directly via svg's
The makefile will be used to deploy a macOS binary or whatever I guess
